src/sim-host/ui/sim-host.html

```diff
@@ -64,24 +64,24 @@
 
     <template id="cordova-radio-template">
         <input id="cordova-radio-template-input" type="radio">
-        <label for="cordova-radio-template-input" class="cordova-radio-label">
+        <label aria-hidden="true" for="cordova-radio-template-input" class="cordova-radio-label">
             <content></content>
         </label>
     </template>
 
     <template id="cordova-checkbox-template">
-        <label for="cordova-checkbox-template-input">
+        <label aria-hidden="true" for="cordova-checkbox-template-input">
             <content></content>
         </label>
         <input id="cordova-checkbox-template-input" type="checkbox">
     </template>
 
     <template id="cordova-label-template">
-        <label></label>
+        <label aria-hidden="true"></label>
     </template>
 
     <template id="cordova-labeled-value-template">
-        <label></label>
+        <div class="spoken-text"><label aria-hidden="true"></label></div>
```

Comment:
This change is reverted because it was primarily for Accelerometer panel where we want to support X(m/s^2). Now that they become edit, this change is not necessary anymore.